archive.conf

```diff
@@ -11,7 +11,7 @@ ARCHIVE_USER=archive
 ARCHIVE_GROUP=archive
 
 # Package extensions
-PKGEXT='.pkg.tar.xz'
+PKGEXT='\.pkg\.tar(\.(gz|bz2|xz|zst|lrz|lzo|Z|lz4|lz))?'
```

Comment:
Can we change this to something that doesn't look like it is a makepkg.conf compatible thing in the places where it is called?

`find "$SCANDIR" -type f -regextype posix-extended -regex ".*$PKGEXT\$"` -- it isn't necessarily obvious that the variable name is completely unrelated to the same variable used elsewhere. In dbscripts, I renamed the variable to `PKGEXTS`.

Reply:
What do you think about `PKGREGEX`? Makes it obvious what it is :P

Reply:
I don't care what color the bikeshed is painted. :) Sounds good to me.